deny.toml

```diff
@@ -20,7 +20,7 @@ allow = [
 
 exceptions = [
     { name = "generational-arena", allow = ["MPL-2.0"] },
-    { name = "unicode-ident", allow = ["MIT", "Apache-2.0", "Unicode-DFS-2016"] },
+    { name = "unicode-ident", allow = ["MIT", "Apache-2.0", "Unicode-DFS-2016", "Unicode-3.0"] },
```

Comment:
I also have a PR which is failing CI due to this issue. In the CI build, I see this:
```
registry+https://github.com/rust-lang/crates.io-index#unicode-ident@1.0.14:4:36
  │
4 │ license = "(MIT OR Apache-2.0) AND Unicode-3.0"
```

Given this, can we remove the `"Unicode-DFS-2016"`?

Reply:
Makes sense. Updated